src/rdkafka_transport.c

```diff
@@ -503,6 +507,13 @@ void rd_kafka_transport_ssl_init (void) {
 	SSL_load_error_strings();
 	SSL_library_init();
 	OpenSSL_add_all_algorithms();
+#else
+#ifdef HAVE_OPENSSL_CRYPTO_THREADID_SET_CALLBACK
+	CRYPTO_THREADID_set_callback(rd_kafka_transport_ssl_threadid_cb);
+#endif
+	OPENSSL_init_ssl(0, NULL);
```

Comment:
This isn't needed according to docs:
https://www.openssl.org/docs/man1.1.0/ssl/OPENSSL_init_ssl.html

Reply:
I removed it.